components/ConvertStepper/Step.js

```diff
@@ -0,0 +1,191 @@
+import React, {useMemo} from 'react'
+import PropTypes from 'prop-types'
+import StatusIcon from './StatusIcon'
+import {keyframes, css} from 'styled-components'
+import TransactionBadge from './TransactionBadge'
+
+const STATUS_DESC = {
+  waiting: 'Waiting for signature',
+  working: 'Transaction being mined',
+  success: 'Transaction confirmed',
+  error: 'An error has occured at the time of transaction',
+}
+
+const spinAnimation = css`
+  mask-image: linear-gradient(35deg, transparent 15%, rgba(0, 0, 0, 1.0));
+  animation: ${keyframes`
+    from {
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform: rotate(360deg);
+    }
+  `} 0.5s linear infinite;
+`
+
+const pulseAnimation = css`
+  animation: ${keyframes`
+    from {
+      opacity: 1;
+    }
+
+    to {
+      opacity: 0.25;
+    }
+  `} 0.75s linear alternate infinite;
+`
+
+const absoluteFill = css`
```

Comment:
Ah that’s a better name than [this one](https://github.com/aragon/convert.aragon.org/blob/master/lib/css-utils.js#L9), I will steal it :laughing:

Reply:
oh I didn't see that there! will update the name and use it instead 😀